packages/gatsby/src/utils/eslint-config.ts

```diff
@@ -17,6 +18,9 @@ export const eslintConfig = (
       extends: [require.resolve(`eslint-config-react-app`)],
       plugins: [`graphql`],
       rules: {
+        // Custom ESLint rules from Gatsby
+        "no-anonymous-exports-page-templates": `warn`,
+        "limited-exports-page-templates": `warn`,
```

Comment:
discussion topic (rather than just applying suggestion)

should those rules (and `jsc-pascal-case`) only be enabled when `fast-refresh` is used?

```suggestion
        "no-anonymous-exports-page-templates": process.env.GATSBY_HOT_LOADER === `fast-refresh` ? `warn` :`off`,
        "limited-exports-page-templates": process.env.GATSBY_HOT_LOADER === `fast-refresh` ? `warn` :`off`,
```

Reply:
PascalCase should always be enabled, we had that before. Outside of fast-refresh it's also helpful.

Until v3 we can use that conditional, yeah 👍

Reply:
Will be handled in https://github.com/gatsbyjs/gatsby/pull/28911

Reply:
To add two cents for support; consistency is good and important.